From 7819dc8b99d2a18f52e6164beeb8f38b93bc628b Mon Sep 17 00:00:00 2001
From: Nobuhiro Iwamatsu <nobuhiro1.iwamatsu@toshiba.co.jp>
Date: Tue, 16 Feb 2021 23:05:09 +0900
Subject: [PATCH 12/12] arm64: dts: visconti: Fix parse error for TMPV7708 RM
 main board

The patch fix commit: c5e188ea08290d9 ("Merge remote-tracking branch
'net-next/master'").
This merge commit fixes tmpv7708-rm-mbrc.dts's conflict, but the typo
causes a parse error. This removes this typo.

ref: https://gitlab.com/Linaro/lkft/mirrors/next/linux-next/-/jobs/1033072509#L382
Reported-by: Naresh Kamboju <naresh.kamboju@linaro.org>
Signed-off-by: Nobuhiro Iwamatsu <nobuhiro1.iwamatsu@toshiba.co.jp>
Signed-off-by: Nathan Chancellor <nathan@kernel.org>
---
 arch/arm64/boot/dts/toshiba/tmpv7708-rm-mbrc.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/toshiba/tmpv7708-rm-mbrc.dts b/arch/arm64/boot/dts/toshiba/tmpv7708-rm-mbrc.dts
index 3760df93a89b..230f487dd6b2 100644
--- a/arch/arm64/boot/dts/toshiba/tmpv7708-rm-mbrc.dts
+++ b/arch/arm64/boot/dts/toshiba/tmpv7708-rm-mbrc.dts
@@ -49,7 +49,7 @@ &wdt {
 
 &gpio {
 	status = "okay";
-};`
+};
 
 &piether {
 	status = "okay";
-- 
2.30.1

